examples/platform/silabs/efr32/wf200/efr_spi.c

```diff
@@ -49,14 +48,24 @@
 #include "sl_power_manager.h"
 #endif
 
-#if SL_WIFI
+#if defined(EFR32MG12)
```

Comment:
This seems unscalable with future platform

Reply:
Shall we introduce something like `SL_ENABLE_MULTIPLEX` ? which is governed by combination of boards and peripherals ?